api/envoy/data/cluster/v2alpha/outlier_detection_event.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package envoy.data.cluster.v2alpha;
+option java_package = "io.envoyproxy.envoy.data.cluster.v2alpha";
+option java_multiple_files = true;
+
+import "google/protobuf/timestamp.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: Health check logging events]
+// :ref:`Outlier detection logging <arch_overview_outlier_detection_logging>`.
+
+message OutlierDetectionEvent {
+
+  OutlierEjectionType type = 1 [(validate.rules).enum.defined_only = true];
+
+  // Timestamp for event.
+  google.protobuf.Timestamp timestamp = 2 [(gogoproto.stdtime) = true];
+
+  int64 secs_since_last_action = 3 [(validate.rules).int64.gte = 0];
+
+  string cluster_name = 4 [(validate.rules).string.min_bytes = 1];
+
+  string upstream_url = 5 [(validate.rules).string.min_bytes = 1];
+
+  string action = 6 [(validate.rules).string.min_bytes = 1];
+
+  int32 num_ejections = 7 [(validate.rules).int32.gte = 0];
+
```

Comment:
WDYT of adding comments or removing most of these intervening blank lines? I looked around other proto files and there are often intervening blank lines but there are also comments for every field.